src/game_renderer/GameRenderer.cpp

```diff
@@ -185,15 +185,244 @@ void GameRenderer::RenderLevelEditor(GLFWwindow* window,
   RenderIt(window, game_state, true);
 }
 
-void GameRenderer::RenderIt(GLFWwindow* window,
+void GameRenderer::RenderMiniMap(GLFWwindow* window,
+                std::shared_ptr<GameState> game_state,
+                bool render_level_editor) {
```

Comment:
almost all of this method is a copy of the main render method, right? could we consolidate it into one method?